### PR TITLE
sql/importer: add hidden setting to adjust procs per node

### PR DESCRIPTION
**Pull request by dt**
Release note: none.

### CODE REVIEW

File: pkg/sql/importer/import_job.go [hunk 1]
```diff
@@ -79,6 +80,13 @@ func (r *importResumer) ForceRealSpan() bool {
 
 var _ jobs.Resumer = &importResumer{}
 
+var processorsPerNode = settings.RegisterIntSetting(
+	settings.TenantWritable,
+	"bulkio.import.processors_per_node",
+	"number of input processors to run on each sql instance", 1,
+	settings.NonNegativeInt,
```
Comment (adityamaru) [inline pkg/sql/importer/import_job.go:87]: should we make this PostiveInt?

File: pkg/sql/importer/import_processor_planning.go [hunk 2]
```diff
@@ -240,16 +241,17 @@ func makeImportReaderSpecs(
...
+	procsPerNode int,
 ) []*execinfrapb.ReadImportDataSpec {
 	details := job.Details().(jobspb.ImportDetails)
 	// For each input file, assign it to a node.
-	inputSpecs := make([]*execinfrapb.ReadImportDataSpec, 0, len(sqlInstanceIDs))
+	inputSpecs := make([]*execinfrapb.ReadImportDataSpec, 0, len(sqlInstanceIDs)*procsPerNode)
 	progress := job.Progress()
 	importProgress := progress.GetImport()
 	for i, input := range from {
 		// Round robin assign CSV files to sqlInstanceIDs. Files 0 through len(sqlInstanceIDs)-1
```
Comment (adityamaru) [inline pkg/sql/importer/import_processor_planning.go:252]: we should update this comment

**Review by adityamaru — APPROVED** [review]

**Comment by dt** [comment]
Screen Shot 2022-04-01 at 8 07 07 AM

File: pkg/sql/importer/import_job.go (hunk 1 above)
Comment (dt) [reply pkg/sql/importer/import_job.go]: Done.

File: pkg/sql/importer/import_processor_planning.go (hunk 2 above)
Comment (dt) [reply pkg/sql/importer/import_processor_planning.go]: Done.

**Comment by dt** [comment]
TFTR!

bors r+

**Comment by dt** [comment]
cc @tbg in case you want to rebase

**Pull request merged**